<?xml version="1.0" encoding="utf-8"?>
<pysyslog completed="4/4" status="complete">
	<timestamp>2022-04-21 14:22:54</timestamp>
	<platform>linux</platform>
	<host>m32sd10a</host>
	<root>/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys</root>
	<xargs>
		<xarg name="myPlatform" value="container"/>
	</xargs>
	<results cycle="1">
		<result id="SoftwareManagement.Plugin.apama_plugin_install" outcome="BLOCKED">
			<outcomeReason>sudo&lt;remove_apama_project&gt; returned non-zero exit code 1 (+1 other failures)</outcomeReason>
			<timestamp>2022-04-21 14:23:02</timestamp>
			<descriptor>/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_install/pysystest.xml</descriptor>
			<output>/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_install/Output/linux</output>
		</result>
		<result id="SoftwareManagement.Plugin.apama_plugin_mon_support" outcome="FAILED">
			<outcomeReason>Assertion that path exists=False for &quot;/etc/tedge/apama/project&quot;</outcomeReason>
			<timestamp>2022-04-21 14:23:33</timestamp>
			<descriptor>/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_mon_support/pysystest.xml</descriptor>
			<output>/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_mon_support/Output/linux</output>
		</result>
		<result id="SoftwareManagement.Plugin.apama_plugin_remove" outcome="PASSED">
			<outcomeReason>Assertion that path exists=False for &quot;/etc/tedge/apama/project&quot;</outcomeReason>
			<timestamp>2022-04-21 14:23:55</timestamp>
			<descriptor>/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_remove/pysystest.xml</descriptor>
			<output>/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_remove/Output/linux</output>
		</result>
		<result id="SoftwareManagement.Plugin.apama_plugin_update" outcome="BLOCKED">
			<outcomeReason>sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+2 other failures)</outcomeReason>
			<timestamp>2022-04-21 14:24:00</timestamp>
			<descriptor>/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_update/pysystest.xml</descriptor>
			<output>/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_update/Output/linux</output>
		</result>
	</results>
</pysyslog>
